tests/test_workflow_api.py

```diff
@@ -0,0 +1,74 @@
+from re import A
+import pytest
+from devtools import debug
+from sqlmodel import select
+
+from fractal_server.app.models import Workflow
+from fractal_server.app.models import WorkflowRead
+from fractal_server.app.models import WorkflowCreate
+from fractal_server.app.models import LinkTaskWorkflow
+
+from fractal_server.app.models import Project
+from fractal_server.app.models import Task
+
+async def test_workflow_post(
+    db, client, MockCurrentUser, project_factory, task_factory
+):
+    async with MockCurrentUser(persist=True) as user:
+        res = await client.post(
+            "api/v1/workflow/",
+            json={"name":"My Workflow"}, 
+        )
+        assert res.status_code==422 # no project_id
+
+        project = await project_factory(user)
+        p_id = project.id
+        workflow = WorkflowCreate(
+            name="My Workflow",
+            project_id=p_id,
+        )
```

Comment:
This is correct. However, it is preferable that you use plain dictionaries to pass data to the endpoints. For example, if in the future we change the `WorkflowCreate` schema, the class might silently add defaults and the test still silently pass even though we need to explicitly pass some extra parameter to it.

a simple change `WorkflowCreate -> dict` will make us a bit safer in the future.